pom.xml

```diff
@@ -28,50 +31,64 @@
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-tomcat</artifactId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
```

Comment:
data-jpa zamiast tomcat? nie bardzo czaje jak to sie do siebie ma. JPA to JAVA Persistence API, a Apache Tomcat to... Java Servlet Container, ktory implementuje tez specyfikacje EE takie jak "Java Servlets, JSP, Java EL, WebSocket"

Reply:
Jest Tomcat, nie bój się:) po prostu GIt pokazał różnice w tym, ze zamieniłem kolejność miejsc w plikach i chyba tego do końca nie wyłapał